js/app.js

```diff
@@ -616,6 +645,30 @@
 
 			OCA.SpreedMe.app.audioDisabled = true;
 		},
+		availableVideo: function() {
+			var $hideVideoButton = $('#hideVideo');
+
+			navigator.getUserMedia = navigator.getUserMedia ||
+                         navigator.webkitGetUserMedia ||
+                         navigator.mozGetUserMedia;
+
+			if (navigator.getUserMedia !== null) {
+				navigator.getUserMedia({ video: true },
+					function() {
+						OCA.SpreedMe.app.enableVideo();
+						localStorage.removeItem("videoDisabled");
+						OCA.SpreedMe.app.videoAvailable = true;
+					},
+		      function(err) {
+						$hideVideoButton.attr('data-original-title', 'No Camera')
```

Comment:
Please make this translateable.

Reply:
ohh i see it, i will also change the other ones.